### PR TITLE
change np.exp to K.exp

### PR DESCRIPTION
**Pull request by the-moliver**
To address https://github.com/farizrahman4u/keras-contrib/issues/127

### CODE REVIEW

File: keras_contrib/layers/normalization.py [hunk 1]
```diff
@@ -319,8 +319,8 @@ def call(self, inputs, training=None):
 
         # update r_max and d_max
         t_val = K.get_value(self.t)
```
Comment (waleedka) [inline keras_contrib/layers/normalization.py]: I believe the real problem is here. K.get_value() runs a TensorFlow session internally to read the value of self.t and returns a float value. This value is 0 when you create the model. And since the call() function is called only once, then t_val will always be zero, and the following calculations will always be based on a t_val value of 0

**Comment by titu1994** [comment]
@waleedka Hi, I am the author of the BatchRenorm code. This was always a hacky solution to simulate time per iteration. 

While the call function was called only once, I added an update to the self.t value (which is updated per batch, thus simulates time step). However, there was another bug where I would constantly update the value in numpy. Since this wasnt in tensorflow graph, the value would stay set to `self.t += float(self.t_delta)`, which turns out to be 1 (since t_delta = 1) and would constantly be updated to same value.

With self.t = 1, this simulates batch renorm behaviour (though not exactly like the paper suggests). This was as close to a fix as I could find : 
```
# update r_max and d_max
r_val = self.r_max_value / (1 + (self.r_max_value - 1) * K.exp(-self.t))
d_val = self.d_max_value / (1 + ((self.d_max_value / 1e-3) - 1) * K.exp(-(2 * self.t)))

self.add_update([K.update(self.r_max, r_val),
                            K.update(self.d_max, d_val),
                            K.update_add(self.t, K.variable(np.array([self.t_delta])))], x)
```

Now, the entire graph is in TF, and the update to self.t is an update_add of a constant value (self.t_delta).

One major change needed for this to work is to reduce t_delta value to 1e-3 or 2e-3. That causes batch renorm to act properly as before.

The fact that the renorm version of WRN-16-4 converges to 61% in the 1st epoch, whereas the batchnorm version only gets 60~% after the first 5 epochs suggests that renorm was working before and will work now as well.

@the-moliver Could you merge these changes in this PR itself? I am travelling and wont be able to create a PR for several days.
